### PR TITLE
Initial commit of Deprecation sample.

### PR DESCRIPTION
**Pull request by rbstarbuck**
This should not be merged in until the morning of 12/6 and should be coordinated with the publishing of the migration guide and blog post.

### CODE REVIEW

File: drive/deprecation/.gitignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+.gradle
+/local.properties
+/.idea/caches/build_file_checksums.ser
+/.idea/libraries
+/.idea/modules.xml
+/.idea/workspace.xml
+.DS_Store
+/build
+/captures
+.externalNativeBuild
```
Comment (grant) [inline drive/deprecation/.gitignore]: Add `.idea/`

File: drive/deprecation/.idea/codeStyles/codeStyleConfig.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```
Comment (grant) [inline drive/deprecation/.idea/codeStyles/codeStyleConfig.xml]: Do not commit `.idea` files.

File: drive/deprecation/.idea/dbnavigator.xml [hunk 3]
```diff
@@ -0,0 +1,454 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```
Comment (grant) [inline drive/deprecation/.idea/dbnavigator.xml]: Ditto.
Do not commit `.idea` files.

File: drive/deprecation/LICENSE [hunk 4]
```diff
@@ -0,0 +1,191 @@
+Apache License
```
Comment (grant) [inline drive/deprecation/LICENSE]: Please remove this.
Do not commit non-top-level LICENSE files.

File: drive/deprecation/app/build.gradle [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        }
+    }
+    buildToolsVersion '28.0.3'
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+    packagingOptions {
+        exclude 'META-INF/DEPENDENCIES'
+        exclude 'META-INF/LICENSE'
```
Comment (grant) [inline drive/deprecation/app/build.gradle]: Are all of these exclude rules needed?

**Comment by grant** [comment]
Can you provide detail to why this change is proposed?

**Comment by rbstarbuck** [comment]
@grant: this is a new sample app demonstrating how users can replicate the Drive Android API's functionality using the Drive REST API. The Android API is being deprecated and clients will need to migrate. This sample will be linked to from a migration guide, blog posts, and an MSA to help developers with the transition.

**Pull request merged**